TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -856,11 +856,14 @@ static control_t test_snprintf_buffer_overflow_generic(const char *fmt, T data)
     char buffer_minimal[buf_size];
     int result_baseline;
     int result_minimal;
-
+//There is a bug in Microlib snprintf with zero byte copy.
+//https://jira.arm.com/browse/SDCOMP-54710
```

Comment:
I would be more explicit about the issue.
// Microlib snprintf always returns zero if the size of the buffer is zero.
// See reported bug https://jira.arm.com/browse/SDCOMP-54710

We will let Martin decide if JIRA reference needs to be removed